### ISSUE TITLE
iPXE won't boot without .0 files

### ISSUE THREAD

**Issue by enoch85** [ltsp/ltsp#64]
Ref: https://github.com/ltsp/ltsp/issues/43#issuecomment-549437889

### Info
```
LTSP version: 19.10-1~201911020603~ubuntu16.04.1

SERVER XSESSIONS: Lubuntu.desktop  Lubuntu-Netbook.desktop  openbox.desktop

SERVER OS: PRETTY_NAME="Ubuntu 16.04.6 LTS"

CHROOTS:


VMs:


IMAGES:
x86_32
```
**Thin client** HP T5745
**Server** VMware 6.5

### Steps to reproduce
1. Installed LTSP according to the documentation.
2. Noticed that dnsmasq made the DNS on the server go away, so I installed all the packages except that one to be able to run `ltsp ipxe`, and then installed dnsmasq again.
3. Made an image and booted the thin client

### Expected behavior
The thin client should boot normally with iPXE

### Actual behavior
The thin client didn't boot since it was requesting `undionly.kpxe.0` and `ltsp.ipxe.0` and those didn't exists since iPXE didn't create them in the tftp root.

Dnsmasq didn't give me DNS in the root shell.

### How I solved it
I simply copied both files `undionly.kpxe` and `ltsp.ipxe` to a new file with a `.0` at the end, and then it booted without any issues.

Regarding dnsmasq I entered `DNSMASQ_EXCEPT=lo` in `/etc/default/dnsmasq`, restarted dnsmasq and now it's working as expected.

**Comment by alkisg** [comment]
Daniel I'm going to close this without any code changes...

The .0 issue is probably a bug in your client PXE stack. Maybe a BIOS update would fix it, for now you can just use symlinks.
Changing the undionly.kpxe and ltsp.ipxe names to .0 just for this broken client doesn't sound like a good solution, but if in the future we see that more client types are affected, then sure, we can change them.

For dnsmasq that was a [bug in Ubuntu16.04](https://github.com/ltsp/ltsp/commit/49123662e5fef16fcde089b7a71cf23882ca073b) which I reported and it was solved in 18.04 with systemd-resolved. I don't think it's worth it to include code in the new LTSP for old-version Ubuntu bugs.

**Comment by enoch85** [comment]
Update: this wasn't an issue in Ubuntu Mate 18.04, only in Lubuntu 16.04, both on i386.
